### PR TITLE
Enhauncement: rework captcha policy

### PR DESCRIPTION
**Pull request by ec**

### CODE REVIEW

**Review by mod — CHANGES_REQUESTED** [review]
Make sure Variable have prefix
BARONG_

values in a single lowercase word
recapcha, geetest, disabled

Make sure there is a documentation with all variables

File: app/api/v2/identity/utils.rb [hunk 1]
```diff
@@ -24,16 +24,16 @@ def language
...
       def verify_captcha!(user:, response:, error_statuses: [400, 422])
-        return if Barong::CaptchaPolicy.config.disabled
+        return if Barong::App.config.barong_captcha == 'disabled'
 
         if response.blank?
           error!({ errors: ['identity.captcha.required'] }, error_statuses.first)
         end
 
-        if Barong::CaptchaPolicy.config.re_captcha
+        if Barong::App.config.barong_captcha == 're_captcha'
```
Comment (mod) [inline app/api/v2/identity/utils.rb]: Spell it recapcha like we do on frontend

File: app/api/v2/identity/utils.rb [hunk 2]
```diff
@@ -24,16 +24,16 @@ def language
...
           recaptcha(user: user, response: response)
         end
-        geetest(response: response) if Barong::CaptchaPolicy.config.geetest_captcha
+        geetest(response: response) if Barong::App.config.barong_captcha == 'geetest_captcha'
```
Comment (mod) [inline app/api/v2/identity/utils.rb]: Spell it geetest like on the frontend

File: config/initializers/barong.rb [hunk 3]
```diff
@@ -43,6 +43,7 @@
   config.set(:barong_geoip_lang, 'en', values: %w[en de es fr ja ru])
   config.set(:barong_session_name, '_barong_session')
   config.set(:redis_url, 'redis://localhost:6379/1')
+  config.set(:barong_captcha, 'disabled', values: %w[disabled recaptcha geetest])
```
Comment (mod) [inline config/initializers/barong.rb]: Validate the possible values or through warning and set to disabled

**Review by mod — CHANGES_REQUESTED** [review]
Make a summary table of variables with 
Name, description, default value, possible values
On the README

File: config/initializers/barong.rb (hunk 3 above)
Comment (ec) [reply config/initializers/barong.rb]: We validate it already

File: app/api/v2/identity/utils.rb (hunk 1 above)
Comment (ec) [reply app/api/v2/identity/utils.rb]: now its 'recaptcha'

**Review by mod — APPROVED** [review]

**Pull request merged**